.github/workflows/static-analysis.yaml

```diff
@@ -17,8 +17,6 @@ jobs:
 
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
-        with:
-          php-version: '7.4'
```

Comment:
I would not remove this, I would just bump the version, or we will get another problem here next november...